Try codepen

Serabe opened the pull request:
This adds a shortcode for including codepens.

dpordomingo commented on layouts/shortcodes/codepen.html:
```diff
@@ -0,0 +1,9 @@
+   data-embed-version="2"
+   data-pen-title="{{ .Get "title" | default "sourced"}}"
+   class="codepen"></p>
+<script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
```
unexpected EOF

dpordomingo replied:
Why not adding the CodePen script in the header as we already do with MathJax?

dpordomingo replied:
**important** 
By the way: `MathJax` script should be moved into [`partial/js.html`](https://github.com/src-d/blog/blob/new-design/themes/hugo-steam-theme/layouts/partials/js.html) instead of being in [`partial/footer.html`](https://github.com/src-d/blog/blob/new-design/themes/hugo-steam-theme/layouts/partials/footer.html#L27), what seems to be wrong

Serabe replied:
I am leaving this as is, because it is the embedded template codepen gives the users. In case it changes, we just need to adjust the shortcode.

dpordomingo replied:
In case the user uses two codepens in the same blog, the script will load twice.
(using the same 3rd party script sometimes fails, could you confirm that codepen does not fail if loaded twice?)

dpordomingo commented on layouts/shortcodes/codepen.html:
```diff
@@ -0,0 +1,9 @@
+   data-theme-id="light"
+   data-slug-hash="{{ .Get "slug"}}"
+   data-default-tab="result"
+   data-user="{{ .Get "user" | default "Serabe" }}"
```
the default should be source{d} user account

dpordomingo commented on content/post/lapjv.md:
```diff
@@ -79,347 +79,7 @@ t-SNE is one of a series of possible dimensionality reduction algorithms which a
-<script src="/post/lapjv/assets/visualize.js"></script>
-<script src="/post/lapjv/assets/figures.js"></script>
-<script src="/post/lapjv/assets/playground.js"></script>
+{{% codepen slug="LjRVRJ" title="lapjv" %}}
```
I would put here the `user="Serabe"`

Serabe commented:
Fixed the account, documented and rebased.

![2017-08-17 at 15 49](https://user-images.githubusercontent.com/36221/29415549-b07768ae-8363-11e7-81d9-74fa8477e618.png)

dpordomingo commented on layouts/shortcodes/codepen.html:
```diff
@@ -0,0 +1,10 @@
+   data-embed-version="2"
+   data-pen-title="{{ .Get "title" | default "sourced"}}"
+   class="codepen"></p>
+<script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
```
In case the user uses two codepens in the same blog, the script will load twice.
(using the same 3rd party script sometimes fails, could you confirm that codepen does not fail if loaded twice?)

Serabe replied:
Tested in person.
